tests/unit/test_met.py

```diff
@@ -215,21 +215,56 @@ def test_wrap_longitude(zero_like_da: xr.DataArray, met_ecmwf_pl_path: str) -> N
 def test_shift_longitude(zero_like_da: xr.DataArray) -> None:
     da = zero_like_da.copy()
 
+    # assign random values to all coords
+    da.loc[:] = np.random.rand(*da.shape)  # noqa: NPY002
```

Comment:
To avoid the [ruff NPY002 error](https://docs.astral.sh/ruff/rules/numpy-legacy-random/), you can do this:

```python
rng = np.random.default_rng()
da.loc[:] = rng.random(*da.shape)
```

It's an extra line of code, but I think it's better practice. If you want to reuse `rng` across different tests, you could define it as a fixture.

Reply:
4e7fbf3cea7942b56b4255ffdd4be4f607406c94